Doc/using/configure.rst

```diff
@@ -190,8 +190,8 @@ Install Options
 Performance options
 -------------------
 
-Configuring Python using ``--enable-optimizations --with-lto`` (PGO + LTO) is
-recommended for best performance.
+Configuring Python using ``--enable-optimizations --with-lto --enable-bolt``
+(PGO + LTO + BOLT) is recommended for best performance.
```

Comment:
Let's be conservative, I would like to introduce the BOLT option as experimental for this time.
I wish that we can change this sentence in the future version.

```suggestion
Configuring Python using ``--enable-optimizations --with-lto``
(PGO + LTO) is recommended for optimal performance.
```

Reply:
I am okay to introduce (PGO + LTO + BOLT) as an experimental combination.  So it is okay to introduce both of them.

Reply:
I agree, also having BOLT installed  is an extra requirement that most users won't have, so I would not recommend to advertise it with the other two options that just depend on the compiler toolchain.

Reply:
Makes sense! What do you think of this new wording?

Reply:
I think is good! Maybe add a link to some more detailed instructions?

Reply:
@pablogsal 

>   BOLT is part of the LLVM project but is not always included in their binary
>   distributions. This flag requires that ``llvm-bolt`` and ``merge-fdata``
>   are available

I think that this sentence is enough. In near future, BOLT will be included in their binary distributions by default.

Reply:
Makes sense. What do you think about adding a link to the BOLT webpage or repo?

Reply:
Good idea, I just added that, let me know what you think.